### PR TITLE
Add Homebrew Cask update to release process

### PR DESCRIPTION
**Pull request by bzz**
Addresses part of the #1654 by adding instructions how to use awesome  [brew bump-cask-pr](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#submit-a-new-version-of-an-existing-formula) automation.

Tested, and the instructions work on both Linux and macOS.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -491,6 +491,13 @@ Then you'll have to create a _Personal access token_ for it in the
 by running any command that does a request to GitHub, like so: `hub api`.
 You'll be asked to provide your GitHub login and the access token.
 
+#### Homebrew CLI
+
+To update a formula at [Homebrew package manager](br), you'll need a
```
Comment (geigerzaehler) [inline DEVELOPMENT.md:496]:
```suggestion
To update a formula at [Homebrew package manager][br], you'll need a
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -491,6 +491,13 @@ Then you'll have to create a _Personal access token_ for it in the
...
+working `brew` CLI tool. The tool can use your GitHub _Persona access token_
```
Comment (geigerzaehler) [inline DEVELOPMENT.md:497]:
```suggestion
working `brew` CLI tool. The tool can use your GitHub _Personal access token_
```

**Review by geigerzaehler — COMMENTED** [review]
Thanks! We’ll give this a try at the next release before we approve it.

File: DEVELOPMENT.md (hunk 1 above)
Comment (bzz) [reply DEVELOPMENT.md]: Thank you, both addressed in 46c8f48

**Comment by NunoAlexandre** [comment]
I will give this a try today :raised_hands:

**Comment by NunoAlexandre** [comment]
PR open https://github.com/Homebrew/homebrew-cask/pull/102515 👀

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -491,6 +491,13 @@ Then you'll have to create a _Personal access token_ for it in the
 by running any command that does a request to GitHub, like so: `hub api`.
 You'll be asked to provide your GitHub login and the access token.
 
+#### Homebrew CLI
+
+To update a formula at [Homebrew package manager][br], you'll need a
+working `brew` CLI tool. The tool can use your GitHub _Personal access token_
+from the previous step with
+`export HOMEBREW_GITHUB_API_TOKEN='<github-api-token>'` for authentification,
+in order to [setup a Cask repository fork, commit and push][bs] on your behalf.
```
Comment (NunoAlexandre) [inline DEVELOPMENT.md:500]:
```suggestion
To update a formula at the [Homebrew package manager][br], you'll need a working 
`brew` CLI tool. 

The `brew`CLI requires a [GitHub Personal access token[gt] to [set up a Cask repository 
fork, commit and push][bs] on your behalf. You can make it available to brew with 
`export HOMEBREW_GITHUB_API_TOKEN='<github-api-token>'`

Note: this Github access token _must_ grant the `public_repo` scope.
```

**Review by NunoAlexandre — COMMENTED** [review]
@bzz thanks a lot for opening this PR.

The process went fairly smooth apart from the part missing to state that this GH Personal token needs to grant the public_repo scope. I have left a suggestion to add that and simply the block a little.

Happy to merge once that's polished 🏄

File: DEVELOPMENT.md (hunk 3 above)
Comment (NunoAlexandre) [reply DEVELOPMENT.md:500]: @bzz let me know if you are happy with the suggestion I left or if you want to change it to your liking. Just don't want to assume your :+1: means green lights and step on your toes.

**Comment by rudolfs** [comment]
Thanks @bzz! As per our [DCO](https://github.com/radicle-dev/radicle-upstream/blob/master/DCO) we'll need you to sign-off on your commits. Would you be able to amend with git commit -s --amend? :)

**Pull request closed without merging**